src/core/modes/walk-mode.ts

```diff
@@ -159,4 +171,25 @@ export class WalkMode implements Mode {
 
     this.decorationTimer = setInterval(updateComboTimerDecoration, 50);
   }
+
+  private getLocation(firstVisibleRange: vscode.Range) {
+    const location = vscode.workspace.getConfiguration('junbae-mode').get('location');
+
+    let visibleStartLine = firstVisibleRange.start;
+    let visibleEndLine = firstVisibleRange.start;
+    const junbaeLocation = { top: '5%', 'font-size': '50px', bottom: 'none' };
+    const timerLocation = { top: '5%', bottom: 'none' };
+
+    if (location === 'bottom') {
+      visibleStartLine = firstVisibleRange.end;
+      visibleEndLine = firstVisibleRange.end;
+      junbaeLocation.top = 'none';
+      junbaeLocation.bottom = '10px';
+      junbaeLocation['font-size'] = '100px';
+      timerLocation.top = 'none';
+      timerLocation.bottom = '70px';
+    }
+
+    return { junbaeLocation, timerLocation, visibleStartLine, visibleEndLine };
+  }
```

Comment:
how about this?

```ts
private getLocation(firstVisibleRange:vscode.Range){
  const location = vscode.workspace.getConfiguration('junbae-mode').get('location');

  switch(location){
    case 'top':
      return { ... }
    case 'bottom':
      return { ... }
    default:
      //NOTE: if remove location config, return 'top' location settings
      return { ... } 
  }
}
```

Reply:
I think it is more complicated

Reply:
It's not complicated. cuz It looks more simply.

if you want to look around of bottom location, you look only `case 'bottom':`
when more cases are, the more readable the code will be.

for Now, we have to move it up and down to look at options. because we modify the object directly and use `let`

Reply:
Using `switch` can give us more readability than using `if` and `let`

Reply:
If additional details arise, a switch case would be good.

Reply:
what is good point of this code (using let and if)?cv

Reply:
The number of conditional statements is small, so 'if' statements are used.

Reply:
I think that's not a good reason for developers. less or many is not a reason between `if` and `switch`.

both (`if` and `switch`) are good now. but it's only valid for now. 
If `switch` is better someday, I think we should use switch from now on.
there is nothing for using `if`

Reply:
If there are few cases, I think the switch statement is more complicated. There are few cases now, but I think the switch statement you mentioned will be correct in the future.